pos_analytic_by_config/tests/test_pos_analytic_by_config.py

```diff
@@ -38,6 +38,9 @@ def setUpClass(cls):
             }
         )
         cls.config = cls.basic_config
+        cls.config.payment_method_ids.filtered_domain(
```

Comment:
Put a comment about why are you doing this.

Reply:
Done!